components/picker/stories/template.js

```diff
@@ -138,7 +138,7 @@ export const Template = ({
 				isReadOnly,
 				customClasses,
 				customStyles: {
-					"display": labelPosition == "left" ? "inline-block" : undefined,
+					"display": labelPosition == "left" ? "inline-flex" : undefined,
```

Comment:
This inline CSS has been removed in #3200 . CSS-977 was marked as "has to be done after" CSS-936 in Jira, so I would wait until that PR merges.

Reply:
Great! I'll keep an eye on that and wait for it to be merged.